### DISCUSSION TITLE
GitHub Education benefits activation

### DISCUSSION THREAD

**Discussion by pi-sen** [community/community#138917] [GitHub Education]
### Hi everyone,

Hi. I was approved for the GitHub Education benefits on 16-Sep. Can you please tell me when can I receive the benefits. Thanks.

**Comment by barishaxxer** [comment]
[How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
dont forget to mark this as answer to close the topic if it answers.

**Comment by davevad93** [comment]
Hi @pi-sen ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @pi-sen was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
